src/services/serverApi.ts

```diff
@@ -90,4 +93,132 @@ export const useGetUsers = () => {
   }
 }
 
+//this is used for getting tags for membersSkillUpdateModal
+export const tagsApi = serverApi.injectEndpoints({
+  endpoints: (builder) => ({
+    //Queries
+    getTags: builder.query({
+      query: () => BASE_URL + '/tags',
+    }),
+    getLevels: builder.query({
+      query: () => BASE_URL + '/levels',
+    }),
+    getSkills: builder.query({
+      query: (username) => `${BASE_URL}/users/${username}/skills`,
+      providesTags: ['Skill'],
```

Comment:
make Skill as a constant as its easy to fall into typos and spend hours debugging them